go.mod

```diff
@@ -1,10 +1,12 @@
 module github.com/cert-manager/csi-driver-spiffe
 
-go 1.22.0
+go 1.23
+
+toolchain go1.23.2
```

Comment:
Dependabot added the `toolchain` directive.
I had to run `make go-tidy` which bumped the go directive.
Probably caused by: https://github.com/cert-manager/csi-lib/pull/67
 * https://github.com/cert-manager/csi-lib/pull/67